raspi-config

```diff
@@ -752,6 +759,59 @@ do_serial() {
   fi
 }
 
+get_bluetooth() {
+  overlay=$(get_config_var dtoverlay $CONFIG)
+  if [ "$overlay" = "0" ]; then
+    echo 0
+  elif [ "$overlay" = "pi3-disable-bt" ]; then
+    echo 1
+  else
+    echo -1 
+  fi
+}
+
+do_bluetooth() {
+  if ! is_pithree; then
+    whiptail --msgbox "Bluetooth configuration is only applicable to Pi3" 20 60 2
+    return 1
+  fi
+  DEFAULT=--defaultno
+  CURRENT=0
+  bluetooth=$(get_bluetooth)
+  if [ $bluetooth -eq -1 ]; then
+    whiptail --msgbox "Device Tree overlay already exists" 20 60 1
+    return 1
+  fi
+  if [ $bluetooth -eq 0 ]; then
+      DEFAULT=
+      CURRENT=1
+  fi
+  if [ "$INTERACTIVE" = True ]; then
+    whiptail --yesno "Would you like Bluetooth to be enabled?" $DEFAULT 20 60 2
+    RET=$?
+  else
+    get_init_sys
+    RET=$1
+  fi
+  if [ $RET -eq $CURRENT ]; then
+    ASK_TO_REBOOT=1
+  fi
+  if [ $RET -eq 0 ]; then
+    clear_config_var dtoverlay $CONFIG
```

Comment:
Thanks. It should not. There is an explicit check to ensure dtoverlay is pi3-disable-bt before clearing it. If different than pi3-disable-bt, it will display and error and not continue.

```
get_bluetooth() {
  overlay=$(get_config_var dtoverlay $CONFIG)
  if [ "$overlay" = "0" ]; then
    echo 0
  elif [ "$overlay" = "pi3-disable-bt" ]; then
    echo 1
  else
    echo -1
  fi
}

do_bluetooth() {
  ...
  bluetooth=$(get_bluetooth)
  if [ $bluetooth -eq -1 ]; then
    whiptail --msgbox "Device Tree overlay already exists" 20 60 1
    return 1
  fi
```